src/views/landing.njk

```diff
@@ -28,8 +28,10 @@
               text: 'Start now',
               isStartButton: true,
               attributes: {
-               id: 'start-now'
-              }
+               id: 'start-now',
+               'data-event-id': 'start-now-button' if piwik,
+               onclick: '_paq.push(["trackGoal", todo-add-goal-id-here]);' if piwik
```

Comment:
Awaiting performance team to create goal in Matomo, and then we can add the goal ID here.